### PR TITLE
Bump guava on tests

### PR DESCRIPTION
**Pull request by jorgebay**
Fix minor security vulnerability.

### CODE REVIEW

File: testing/pom.xml [hunk 1]
```diff
@@ -165,7 +165,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>19.0</version>
+      <version>29.0-jre</version>
```
Comment (tomekl007) [inline testing/pom.xml:168]: I see in a lot of places in code that we are using shaded`com.datastax.oss.driver.shaded.guava` and that was a deliberate decision. In that case, why do we even need to declare the guava dependency expliclity?

Comment (jorgebay) [reply testing/pom.xml:168]: Probably different uses through shaded and direct dependencies and have grown organically in the code, without much attention.

In this case, I'm trying to [fix this SV](https://github.com/stargate/stargate/network/alerts).
We can create a different ticket for unifying dependencies.

**Review by tomekl007 — APPROVED** [review]

File: testing/pom.xml (hunk 1 above)
Comment (tomekl007) [reply testing/pom.xml:168]: Ok, maybe we should create a dedicated JIRA ticket for it?

Comment (jorgebay) [reply testing/pom.xml:168]: yeah (we are using GH issues now)

**Pull request merged**